app/services/proofing/lexis_nexis/ddp/verification_request.rb

```diff
@@ -0,0 +1,45 @@
+module Proofing
+  module LexisNexis
+    module Ddp
+      class VerificationRequest < Request
+        private
+
+        def build_request_body
+          {
+            api_key: config.api_key,
+            org_id: config.org_id,
+            account_address_street1: applicant[:address1],
+            account_address_street2: applicant[:address2] || '',
+            account_address_city: applicant[:city],
+            account_address_state: applicant[:state],
+            account_address_country: 'US',
+            account_address_zip: applicant[:zipcode].match(/^\d{5}/).to_s,
+            account_date_of_birth: Date.parse(applicant[:dob]).strftime('%Y%m%d'),
```

Comment:
Date.parse can raise an exception if it's not a string in the right format (have seen similar issues with other proofers), should we guard against that?

Reply:
I think we should definitely make this safe if the dob is nil/missing, but I think we've been trying to make sure that our DOBs are parsed consistently out of vendors... like my hope is that it doesn't get this far in our process if the DOB is formatted incorrectly

Reply:
Yes.  I was expecting a clean dob at this point since it's required previously.  I guess I could throw in an empty check